quirks/websites-with-shared-credential-backends.json

```diff
@@ -217,5 +217,9 @@
     [
         "lookmark.io",
         "lookmark.link"
+    ],
+    [
+        "playstation.com",
+        "id.sonyentertainmentnetwork.com"
```

Comment:
Are there login pages served from both of these domains that accept the same credentials?

Reply:
playstation.com will forward you to sony.com if you want to sign in

Reply:
Then maybe this PR isn’t valid or needed.

Reply:
This was just a fix for issue #75, feel free to close if not valid.

Reply:
I appreciate you giving this a go, @nicklasholm. :)